packages/client/hmi-client/src/workflow/ops/dataset/tera-dataset-drilldown.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<section tabName="Description">
+			<tera-drilldown-section :is-loading="fetchingDataset">
+				<div>
+					<tera-dataset-description :dataset="dataset" :raw-content="rawContent" />
+				</div>
+			</tera-drilldown-section>
+		</section>
+		<section tabName="Data">
+			<tera-drilldown-section :is-loading="fetchingDataset">
+				<div class="datatable-container">
+					<tera-dataset-datatable :rows="100" :raw-content="rawContent" />
+				</div>
```

Comment:
remove div and move this container class to the component

Reply:
done